plasmapy/particles/decorators.py

```diff
@@ -9,7 +9,7 @@
 from collections.abc import Callable, Iterable, MutableMapping
 from inspect import BoundArguments
 from numbers import Integral, Real
-from typing import Any, Optional, TypedDict, Union
+from typing import Any, Optional, TypedDict, Union, get_type_hints
```

Comment:
ChatGPT suggested using `typing.get_type_hints` because other methods returned the string representations of the annotations rather than the actual objects.  Probably saved me an hour or two of work...

Reply:
ChatGPT :scream: